rails-time-track/db/seeds.rb

```diff
@@ -5,3 +5,72 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require 'faker'
+
+3.times do
```

Comment:
Maybe we can create only 1 owner. What do you think?

Reply:
Thanks. I have changed that.